test/bitfields_packed.ll

```diff
@@ -0,0 +1,125 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv --spirv-debug-info-version=ocl-100 %t.bc
+; RUN: llvm-spirv --spirv-ext=+SPV_KHR_non_semantic_info --spirv-debug-info-version=nonsemantic-shader-100 %t.bc
+; RUN: llvm-spirv --spirv-debug-info-version=nonsemantic-shader-200 %t.bc -o %t.spv
+; RUN: llvm-spirv %t.spv -o %t.spt --to-text
+; RUN: llvm-spirv -r -emit-opaque-pointers %t.spv -o %t_1.bc
+; RUN: llvm-dis %t_1.bc -o %t_1.ll
+; RUN: FileCheck %s --input-file %t.spt -check-prefix=CHECK-SPIRV
+; RUN: FileCheck %s --input-file %t_1.ll  -check-prefix=CHECK-LLVM
+
+;CHECK-SPIRV: String [[#A:]] "a"
+;CHECK-SPIRV: String [[#B:]] "b"
+;CHECK-SPIRV: String [[#C:]] "c"
+;CHECK-SPIRV: String [[#D:]] "d"
+;CHECK-SPIRV: String [[#E:]] "e"
+;CHECK-SPIRV: String [[#F:]] "f"
+
+;CHECK-SPIRV: Constant [[#]] [[#]] 3
```

Comment:
we should also check of constant 262144
and following DebugTypeMember should contain it

Reply:
I do not see this in the .spt file. Is this related to the 'scope' issue you mentioned later? Thanks